locales/en/app.ftl

```diff
@@ -334,16 +340,19 @@ user-fb-compromised-single =
   This breach exposed the following personal info. If you haven’t already, 
   change your passwords.
 user-generic-fb-compromised-single = This breach exposed the following personal info.
-guest-fb-compromised-single = 
-  This breach exposed the following personal info. Sign up for a {-brand-fxa} 
-  to get your full report and alerts about new breaches.
 
-guest-fb-compromised-blurb =
+guest-fb-compromised-single-v2 = This breach exposed the following personal info. 
+    Create a free {-brand-fxa} for your full report of past breaches, new breach alerts, 
+    and info about other {-brand-Mozilla} services.
```

Comment:
Note to self: I've overlooked this when we created terms, we should be consistent and use kebab-case for message IDs (in the future).

Reply:
> and use kebab-case for message IDs (in the future).

It looks like only a handful of mixed-Case terms, if it's something you think we should change:

```
Term => brand-Quantum
Term => brand-Mozilla
Term => brand-HIBP
Term => brand-Chrome
Message => layout-Firefox
```

<details>
<summary>Source</summary>

```js
const fs = require("fs");
const { parse } = require("fluent-syntax");

const argv = process.argv.slice(2);

checkLocale(...argv);

function checkLocale(ftlFile = "./locales/en/app.ftl") {
  const ftl = fs.readFileSync(ftlFile, "utf-8");
  const { body } = parse(ftl, { withSpans: false });
  const checkCase = str => str === str.toLowerCase();

  body.forEach(resource => {
    switch (resource.type) {
      case "Term":
      case "Message":
        if (!checkCase(resource.id.name)) {
          console.log(`${resource.type} => ${resource.id.name}`);
        }
        break;
    }
  });
}
```
</details>

Reply:
As said, it's something to keep in mind for the future. Changing any of these would invalidate all existing strings.